package.json

```diff
@@ -55,25 +55,25 @@
   },
   "dependencies": {
     "debug": "^4.1.1",
-    "minimist": "^1.2.0",
+    "minimist": "^1.2.8",
     "pump": "^3.0.0",
-    "rotating-file-stream": "^3.0.0",
+    "rotating-file-stream": "^3.1.0",
     "split2": "^4.0.0",
```

Comment:
Unless there is a required security patch, these ranges should remain as is for compatibility.